### PR TITLE
docs: Update get-started.md

### PR DESCRIPTION
**Pull request by Borges-Porteous**

### CODE REVIEW

File: .bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"
```
Comment (delisma) [inline .bundle/config:2]: @Borges-Porteous Was that created by the `bundle` command or you created it yourself?

Comment (Borges-Porteous) [reply .bundle/config:2]: Was automatically done, no idea

**Comment by Borges-Porteous** [comment]
Question for you @delisma now that I've made some updates, how should I proceed to share the preview with colleagues and get their input? I guess I have to push it back to gh-pages to get it working live? (other than just changing the branch making the github preview...)

File: .bundle/config (hunk 1 above)
Comment (delisma) [reply .bundle/config:2]: @Borges-Porteous can you add `.bundle` in the `.gitignore` file please?

**Comment by delisma** [comment]
> Question for you @delisma now that I've made some updates, how should I proceed to share the preview with colleagues and get their input? I guess I have to push it back to gh-pages to get it working live? (other than just changing the branch making the github preview...)

@Borges-Porteous This is where the Netlify workflow would shine. Unfortunately we can only publish one branch on GitHub but on Netlify any branch can be published. Otherwise we all need to sync up the repo to get your changes locally and than serve on our individual machine, which I think is possible.
/cc @nschonni 

**Update**
So it's seems that GitHub get this question frequently so they add an alert to provide the answer
> image

`Check out locally`
```
git fetch origin
git checkout gh-pages-community
git merge --ff-only @{upstream}
```
/cc @quidampepin

**Review by quidampepin — APPROVED** [review]
Looks good to me as a starting point.

**Review by delisma — CHANGES_REQUESTED** [review]
@Borges-Porteous Add this line in your `.gitignore` file under the Jekyll site files
`.bundle`

**Comment by nschonni** [comment]
General note, since you're working with source control now, it might be better to delete the sections rather than commenting them out. The can be pulled back out of the history later if you need them

**Comment by Borges-Porteous** [comment]
> General note, since you're working with source control now, it might be better to delete the sections rather than commenting them out. The can be pulled back out of the history later if you need them

Haha, you're really paying attention to the changes! Sounds good, 

I was just hesitant with these particular sections since rather than delete them, they probably should exist on a different section: we need the latest updates somewhere, but in the get started section might not be the place.

**Comment by Borges-Porteous** [comment]
Note that I've merged it back into gh-pages. Since the gh-pages is draft anyway, need it there to share with the team along with the community pages that David is going to share.

We can improve our branch draft process with Netlify... looking forward to that.

**Pull request merged**